src/wwwroot/app.js

```diff
@@ -123,8 +123,8 @@
           controller: 'DomainManagerCtrl',
           controllerAs: 'vm'
         })
-        .when('/settings/dkim-configuration-help', {
-          templateUrl: 'partials/settings/dkim-configuration-help.html',
+        .when('/settings/domain-manager/dkim-configuration-help', {
+          templateUrl: 'partials/settings/domain-manager/dkim-configuration-help.html',
```

Comment:
What does happen with the old URL? I mean, suppose that someone has created a bookmark or if we are exposing the old URL in some documentation post. Please, ensure to redirect old URL to the new one.

Reply:
💤 💤 💤

Reply:
@mvillaseco 💤 💤 💤

Reply:
Fixed (I think) in #202

Reply:
It is supposed that this page should be accessed directly, so I will revert the "fix".

If someone has a bookmark, it will be broken, but I think that no one has a bookmark.